documentation/address_space_admin/master.adoc

```diff
@@ -0,0 +1,16 @@
+include::common/attributes.adoc[]
+
+= {Title}
+
+include::modules/.adoc[leveloffset=+1]
+
```

Comment:
should these just be commented out (we have no modules sub directory, and obviously we wouldn't expect a doc called just ".adoc")

Reply:
@rgodfrey, yes, those were intended solely as stubs. Will comment it out